### PR TITLE
CLN: Removed duplicated test data

### PR DESCRIPTION
**Pull request by alimcmaster1**
Remove duplicated test data.

"test1.csv" & "tips.csv" are present in both "tests/io/parser/data" & "tests/io/data/csv".

Move the "*.csv.bz2" & "*.csv.gz" files since the s3_resource fixture in tests/io/conftest requires these.

Follow up from: https://github.com/pandas-dev/pandas/issues/29439

Think we should move all the data in io/parser/data/* into io/data/ so it can be re-used. Will save for follow up.

### CODE REVIEW

**Comment by MarcoGorelli** [comment]
Hey @alimcmaster1 

Is part of this already addressed in #34458?

**Comment by alimcmaster1** [comment]
> Hey @alimcmaster1
> 
> Is part of this already addressed in #34458?

Morning, thanks for pointing this out. I think we will just want to remove pandas/tests/data in that PR? Whereas here I was looking at io/parser/data and io/data directories here

**Comment by alimcmaster1** [comment]
@WillAyd you able to review this? Thanks!

File: pandas/tests/io/parser/conftest.py [hunk 1]
```diff
@@ -53,11 +53,11 @@ def csv_dir_path(datapath):
 
 
 @pytest.fixture
-def csv1(csv_dir_path):
```
Comment (WillAyd) [inline pandas/tests/io/parser/conftest.py:56]: What's the reason for doing this instead of editing `csv_dir_path`?

Comment (alimcmaster1) [reply pandas/tests/io/parser/conftest.py:56]: csv_dir_path is used across quite a few tests. We would have to move all csvs from io/parser/data to io/data/csv if we wanted to do this.

Comment (WillAyd) [reply pandas/tests/io/parser/conftest.py:56]: Gotcha. I think makes sense to do that but can be done as a follow up

**Review by WillAyd — APPROVED** [review]
lgtm @jreback

**Comment by jreback** [comment]
thanks @alimcmaster1 very nice

**Pull request merged**